tests/behavior/transforms/pipelines_complete.toml

```diff
@@ -10,7 +10,7 @@ address = "0.0.0.0:9000"
 [transforms.processing]
 type = "pipelines"
 inputs = ["datadog_agent", "syslog"]
-mode = "linear"
+mode = "serial"
```

Comment:
I'm having a little trouble following where we use "linear"/"route" vs "serial"/"parallel, but it seems like this should be "linear" because it's the user-facing pipeline mode?